server/src/main/java/fr/uga/l3miage/tp2/exo1/models/developerEntity.java

```diff
@@ -0,0 +1,21 @@
+package fr.uga.l3miage.tp2.exo1.models;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import java.util.Set;
+
+@Entity
+@Table(name="developer_miage")
+public class developerEntity {
+    @Id
+    private String lastName;
+
+    private String firstName;
+
+    private String email;
+
+    @ManyToMany(mappedBy = "developers")
+    private Set<ProjectEntity> projets;
```

Comment:
```suggestion
    @ManyToMany
    private Set<ProjectEntity> projets;
```
votre relation est à l'envers, le `mappedBy` devrait être dans l'entité `ProjectEntity` car la actuellement vous avez une table d'association qui ce nomme `projects_developper_miage` alors que je voulais une table qui commence `developper_miage_projects`